src/pyramid/config/routes.py

```diff
@@ -283,6 +283,8 @@ def add_route(
 
           .. versionadded:: 1.4a4
 
+          .. deprecated:: 2.0
```

Comment:
The docs build, but I haven’t checked the output.  I will compare with the method below, where the whole doc block is inside the deprecated directive.

Reply:
Tested. Output looks fine.